app/assets/javascripts/quby/answers/validation.js

```diff
@@ -189,6 +208,48 @@
     return !failed;
   };
 
+  function parsePartialDate(inputs) {
+    var value = function(placeholder) {
+      var val = inputs.filter("[placeholder=" + placeholder + "]").first().val();
+      if(val === undefined || val == "")
```

Comment:
Kun je dit laatste comment nader toelichten? Ik begrijp niet precies wat het probleem is. Gaat de validatie stuk of kan deze netter?